Firmware/mmu2.h

```diff
@@ -283,6 +280,17 @@ class MMU2 {
     /// @returns false if the MMU is not ready to perform the command (for whatever reason)
     bool WaitForMMUReady();
 
+    /// Redundancy test. After MMU completes a tool-change command
+    /// the printer will retract the filament by a distance set by the
+    //  Extra Loading Distance MMU register. If the Fsensor untriggers
```

Comment:
This comment isn't right. The test load should feed forward and then retract a differential amount.